imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -1146,9 +1164,9 @@ func Test_fdReaddir(t *testing.T) {
 					},
 				}
 			},
-			buf: 0, bufLen: 26, // length is long enough for one more.
-			cookie:          2,  // previous d_next
-			expectedBufused: 26, // length to read exactly one more.
+			buf: 0, bufLen: 26, // length is long enough for exactly second.
+			cookie:          1,  // d_next of first
```

Comment:
I have changed cookie 2 to 1, because when you send in cookie 1, you expect the second entry.